ansible/aodn-database/roles/common/files/imos--1.0.sql

```diff
@@ -0,0 +1,391 @@
+-- Harvester support
```

Comment:
Need to have a process for managing updates to this code (we can't do this at the moment and its causing issues) - liquibase?

Reply:
Yes, it would be great to manage properly somehow. This is just pasted in for POC.

Reply:
No probs - TODO item